client/src/components/AuthProvider.jsx

```diff
@@ -5,35 +5,43 @@ export const AuthContext = createContext();
 
 // Create our Provider (wrapper component)
 const AuthProvider = ({ children }) => {
-  const [user, setUser] = useState({ id: null, username: "Guest" });
+  const [user, setUser] = useState({
+    id: 1,
+    username: "Guest",
+  });
   const [loggedIn, setLoggedIn] = useState(false);
+  const [cart, setCart] = useState({
+    id: null,
+    userId: 1,
+    totalPrice: null,
+    items: [],
+  });
 
   useEffect(() => {
     async function getFetchMe() {
       try {
         const result = await fetchMe();
-        
-        if (result.success) {
-          
+
+        if (result.success && result.user.id > 1) {
```

Comment:
i think the Database id start at 1, so the user with an id of 1 is always guest.